OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -604,9 +599,54 @@ public void RemovedFromWorld(Actor self)
 			self.World.ActorMap.RemoveInfluence(self, this);
 			self.World.ActorMap.RemovePosition(self, this);
 			self.World.ScreenMap.Remove(self);
-			IsAirborne = false;
+			OnAirborneAltitudeLeft();
+			OnCruisingAltitudeLeft();
```

Comment:
nit: These two should be flipped in case future `OnCruisingAltitudeLeft()` behavior assumes airborne.